src/main/java/edu/uade/apd/tpo/controller/SistemaDeposito.java

```diff
@@ -13,19 +18,44 @@
 
     private ArticuloDao articuloDao;
     private ItemLoteDao itemLoteDao;
+    private PosicionDao posicionDao;
 
     private static SistemaDeposito instance;
 
     private SistemaDeposito() {
         this.articuloDao = ArticuloDao.getInstance();
         this.itemLoteDao = ItemLoteDao.getInstance();
+        this.posicionDao = PosicionDao.getInstance();
     }
-
-    public static SistemaDeposito getInstance() {
-        if (instance == null) {
-            instance = new SistemaDeposito();
-        }
-        return instance;
+    
+    public void liberarPosicion(String codigoUbicacion, int cantidad) {
+    	Posicion posicion = posicionDao.findById(codigoUbicacion);
+    	posicion.liberar(cantidad);
+    	posicionDao.save(posicion);
```

Comment:
Hay que llamar a posicion.guardar() y dentro de posicion.guardar() se llama a posicionDao.getInstance().save(this).-
Tomar de referencia el SistemaFacturacion y la Factura.-